src/templates/partials/nft-cards.njk

```diff
@@ -1,4 +1,28 @@
-<img class="nft-card__image" src="src\images\static\nft-card-cat.webp" alt="NFT Item 2">
-    <span class="nft-card__badge nft-card__badge--new">NEW</span>
-        <p class="nft-card__price">$199</p>
-        <h3 class="nft-card__title">NAME PRODUCT</h3> 
\ No newline at end of file
+<section>
+    <div>
+      <h2>Catalog</h2>
+      <div>
+        <a href="#">SEE ALL</a>
+        <div>
+          <button>&#8592;</button>
+          <button>&#8594;</button>
+        </div>
+      </div>
+    </div>
+    <div>
+      <ul>
+        {% for nft in nftCards %}
```

Comment:
I added example for data.json and if you will use it, your code:
`{% for nft in **cards** %}`
when **cards** is a name of array from data.json

Reply:
But, if you will use include inside cycle, it will be change to:
`{% for nft in cards %}`

And in `nft-card.njk` use **item** instead of **nft**.

```
<img class="nft-card__image" src="{{ item.image }}" alt="NFT Item 1">
    <span class="nft-card__badge nft-card__badge--{{ item.state }}">{{ item.label }}</span>
    <p class="nft-card__price">{{ item.price }}</p>
    <h3 class="nft-card__title">{{ item.name }}</h3>
```